applications/crossbar/doc/user_authentication.md

```diff
@@ -12,9 +12,9 @@ Provides an auth-token via user credentials
 
 Key | Description | Type | Default | Required | Support Level
 --- | ----------- | ---- | ------- | -------- | -------------
-`account_name` | The account name of the user | `string(1..128)` |   | `false` |  
+`account_name` | The account name of the user | `string(1..128)` |   | `true` |  
 `account_realm` | The account realm of the user | `string(4..253)` |   | `false` |  
-`credentials` | A hash of the uses credentials | `string(1..64)` |   | `true` |  
+`credentials` | A hash of the user credentials | `string(1..64)` |   | `true` |  
```

Comment:
This should be fixed in crossbar/priv/couchdb/schemas/user_auth.json, then run `make apis` from the root which should auto-update this table.